airbyte-integrations/connectors/source-stripe/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk==0.52.8", "stripe==2.56.0", "pendulum==2.1.2"]
+MAIN_REQUIREMENTS = ["airbyte-cdk>=0.53.4", "stripe==2.56.0", "pendulum==2.1.2", "pyrate-limiter~=3.1.0"]
```

Comment:
need to update this after merging https://github.com/airbytehq/airbyte/pull/32411 and releasing the cdk